### PR TITLE
Feature: macOS, watchOS, tvOS support

### PR DESCRIPTION
**Pull request by Brett-Best**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -14,7 +19,7 @@ let package = Package(
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(url: "https://github.com/CodeSlicing/pure-swift-ui.git", .exact("2.0.1")),
+        .package(url: "https://github.com/Brett-Best/pure-swift-ui.git", .branch("feature/MPP-Support")),
```
Comment (Brett-Best) [inline Package.swift]: This will need changing once https://github.com/CodeSlicing/pure-swift-ui/pull/3 is merged.

**Review by Brett-Best — COMMENTED** [review]
@CodeSlicing some of the colour logic could probably be expanded to use NSColor / be refactored a bit.

**Comment by Brett-Best** [comment]
Would love to see https://github.com/CodeSlicing/pure-swift-ui-tools/pull/1 also (:

File: Package.swift (hunk 1 above)
Comment (CodeSlicing) [reply Package.swift]: Before merging can you change this back to CodeSlicing and .branch("feature/3.0.0") please.

**Pull request merged**